taprpc/mintrpc/mint.proto

```diff
@@ -33,40 +33,74 @@ service Mint {
     rpc ListBatches (ListBatchRequest) returns (ListBatchResponse);
 }
 
-message MintAsset {
+message MintSeedling {
```

Comment:
Is this the first time we're exposing users to the concept of a "seedling"? Perhaps we should add a doc comment to explain what this message type is.

I don't think we should hold up this PR on terminology. But perhaps we should think carefully about "seepling" if this is the first time a user might encounter that word in this context. `MintCandidateAsset`?

Reply:
IIUC, this will also trigger a larger breaking change as well (proto compilation failing). Seems non-essential?

Reply:
Looks like just for display, could be `PendingBatchAsset`, or just `PendingAsset`.